app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/cases_list.scala.html

```diff
@@ -31,20 +31,20 @@
         </tr>
     </thead>
     <tbody class="govuk-table__body">
-        @for(c <- cases) {
+        @for((c: Case, index: Int) <- cases.zipWithIndex) {
             @defining(c.application) { a: Application =>
                 @if(a.isBTI) {
                     @defining(a.asBTI) { btiApp: BTIApplication =>
-                        <tr class="govuk-table__row">
+                        <tr id="cases_list-row-@index" class="govuk-table__row">
                             <th class="govuk-table__header" scope="row">
-                                <a href="@uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.CaseController.summary(c.reference)">@c.reference</a>
+                                <a id="cases_list-row-@{index}-reference" href="@uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.CaseController.summary(c.reference)">@c.reference</a>
```

Comment:
A tiny thing. This line has slightly different syntax to line 38. Do we need the curly brackets here?